src/Parser/VengeanceDemonHunter/CPM_ABILITIES.js

```diff
@@ -8,63 +14,81 @@ export const SPELL_CATEGORY = {
 const CPM_ABILITIES = [
   {
     spell: SPELLS.SPIRIT_BOMB_TALENT,
-    isActive: combatant => combatant.lv90Talent === SPELLS.SPIRIT_BOMB_TALENT.id,
+    isActive: combatant => combatant.hasTalent(SPELLS.SPIRIT_BOMB_TALENT.id),
     category: SPELL_CATEGORY.ROTATIONAL,
     getCooldown: haste => 18,
     recommendedCastEfficiency: 1.0,
+    extraSuggestion: <span><br></br>This is your core healing ability. Try to refresh it even if you have just one <SpellLink id={SPELLS.SOUL_FRAGMENT.id} /> available. </span>,
```

Comment:
Is the linebreak really needed? The suggestions being at the end of the line has been intended and is the case anywhere else.